Apsitvarkom.Api/Apsitvarkom.Api.csproj

```diff
@@ -7,7 +7,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
+    <PackageReference Include="AutoMapper" Version="11.0.1" />
+    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="11.0.0" />
```

Comment:
Is it possible to have `AutoMapper` package in `Apsitvarkom.Models` project since there's where all of our mappings will be located? And of course keep `AutoMapper.Extensions.Microsoft.DependencyInjection` in `Apsitvarkom.Api`, since it's needed to register mapping profiles to DI.

Reply:
Yes, that is possible. I have excessively left the package in the DataAccess project too, as I wasn't sure whether we should generate already mapped results or not, so I reverted it now.